bin/ch/Helpers.cpp

```diff
@@ -133,9 +58,9 @@ HRESULT Helpers::LoadScriptFromFile(LPCSTR filename, LPCWSTR& contents, bool* is
     //
     // Read the entire content as a binary block.
     //
-    fread((void*)contentsRaw, sizeof(char), lengthBytes, file);
+    fread(pRawBytes, sizeof(BYTE), lengthBytes, file);
     fclose(file);
-    *(WCHAR*)((byte*)contentsRaw + lengthBytes) = _u('\0'); // Null terminate it. Could be LPCWSTR.
+    *reinterpret_cast<WCHAR*>(pRawBytes + lengthBytes) = 0; // Null terminate it. Could be UTF16
```

Comment:
> You mean char16_t isn't guaranteed to be 2-bytes? I can't believe... What's the use of char16_t if it doesn't guarantee to be `16 bits"?

Logical answer: What if that particular platform min. 32 bits?

C++ International Standard: ...in particular, sizeof(bool), sizeof(char16_t),sizeof(char32_t), and sizeof(wchar_t) are implementation-defined. Guess why?

 http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2013/n3690.pdf